src/tasks/pollYoutube.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-console */
+
+const { schedule } = require('node-cron');
+const fetchLatestYoutubeVideos = require('../helpers/fetchData');
+const Video = require('../api/video/video.model');
+
+// 59 * * * * - every hour.
+// * * * * * - every minute, just for testing porpuses
+
+schedule('* * * * *', async () => {
+  try {
+    const { date: lastDate } = await Video.findOne({}).sort({ createdAt: -1 });
+    // lastDate is the lasted upload video date registered in DB
+    const fetchedVideos = await fetchLatestYoutubeVideos({ publishedAfter: lastDate.toISOString() });
+
+    if (fetchedVideos.length > 0) {
+      fetchedVideos.forEach(async newVideo => {
+        const video = new Video(newVideo);
+        await video.save((err, res) => {
```

Comment:
This line can be combined with the line above and you shouldn't need the callback. (See how it was done in the faq controller)